ruby-challenges.rb

```diff
@@ -10,18 +10,76 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+# Pseudocode: 
+    # Define a function named 'includes_this_letter' that has 2 parameters 'array_of_words' and 'letter'
+    # Create an empty array named 'our_result' to hold the words that contain the 'letter' we are looking for 
+    # Iterate over each word in the given array by using .each
+    # Check if each word contains the letter in our given variables by using the .include? method
+    # If it does have the letter in our given variable, the string will be added to 'our_result' using <<
+    # The return will consist of the array containing the words that include the given letters
+
+        def includes_this_letter(array_of_words, letter)
+            our_result = []
+            array_of_words.each do |word|
+            if word.include?(letter)
+                our_result << word
+            end
+        end
+            return our_result
+        end
+
```

Comment:
Nice! Bonus challenge: check out `.select` and see if you can refactor so that you don't need an "if" statement and the `our_result` variable at all!